<?xml version="1.0" ?>

<launch>
	
	<!--<remap from="/rover/cmd_vel" to="/navigation_velocity_smoother/raw_cmd_vel"  />-->
	<remap from="/rover/cmd_vel" to="/cmd_vel"  />
	
	<include file="$(find rover_ctrl_interface)/launch/tf.launch" />	
	<include file="$(find rover_ctrl_interface)/launch/rover_sensors.launch" />
	
	<!-- Joypad -->
	<node name="joy" pkg="joy" type="joy_node" output="screen">
		<param name="dev" type="string" value="/dev/input/js0"/>
	</node>

	<node pkg="rover_ctrl_interface" type="rover_interface" name="rover_interface" output="screen">

		<param name="mushroom_joy" type="bool" value="true" />
		<param name="control_motors" type="bool" value="true" />
		<param name="joy_vel_ctrl" type="bool" value="true" />

		<param name="ctrl_rate" value="100" type="int" />
		<param name="lin_vel_joy_axes" value="1" type="int" />
		<param name="ang_vel_joy_axes" value="3" type="int" />
		<param name="mush_joy_but" value="4" type="int" />

		<param name="max_lin_vel" value="0.5" type="double" />
		<param name="max_ang_vel" value="0.4" type="double" />

		<param name="publish_tf" value="true" type="bool" />
		<param name="rotation_from_imu" value="true" type="bool" />

	</node>
</launch>

